src/screens/ProfileScreen.js

```diff
@@ -1,33 +1,34 @@
-import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, Text, View, Button, Dimensions, TouchableOpacity } from 'react-native';
-import { authentication } from '../../firebaseConfig';
-import { signOut } from "firebase/auth"
-import useAuthentication from '../utils/useAuthentication';
-import Header from '../components/Header';
-import { globalStyle, colors } from '../styles/styles';
+import { StatusBar } from 'expo-status-bar'
+import {
+  StyleSheet,
+  Text,
+  View,
+  Dimensions,
+  TouchableOpacity,
+} from 'react-native'
+import { authentication } from '../../firebaseConfig'
+import { signOut } from 'firebase/auth'
+import useAuthentication from '../utils/useAuthentication'
+import Header from '../components/Header'
+import { globalStyle, colors } from '../styles/styles'
 
 /** Screen where the profile can be viewed. */
-const ProfileScreen = ( {navigation} ) => {
-  const { user } = useAuthentication();
+const ProfileScreen = ({ navigation }) => {
+  useAuthentication()
```

Comment:
We will definitely need user here

Reply:
lint will yell at you here so when we need it we can put it back